datumaro/plugins/searcher.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import logging as log
+import os.path as osp
+
+import numpy as np
+import torch
+from PIL import Image
+from torchvision import transforms
+
+from datumaro.components.model_inference import compute_hash, download_file, tokenize
+from datumaro.plugins.openvino_plugin.launcher import OpenvinoLauncher
+
+
+class SearcherLauncher(OpenvinoLauncher):
+    def __init__(
+        self,
+        description=None,
+        weights=None,
+        interpreter=None,
+        device=None,
+        model_dir=None,
+        output_layers=None,
+    ):
+        model_name = "clip_visual_ViT-B_32"
+        if description or weights:
+            model_name = osp.splitext(description)[0]
+
+        url_folder = "http://s3.toolbox.iotg.sclab.intel.com/test/components-data/datumaro/models/"
+        if not model_dir:
+            model_dir = "tests/assets/searcher"
```

Comment:
Upload this model file to https://storage.openvinotoolkit.org/ and replace the model path in case of `model_dir == None` with a commonly used directory path, e.g. `~/.cache/datumaro`. Please refer to https://stackoverflow.com/questions/7401256/good-way-and-or-platform-independent-place-to-cache-data-from-web.

Reply:
I changed to use `~/.cache/datumaro` folder instead of `tests/assets/searcher`. After I uploaded https://storage.openvinotoolkit.org/, I will update to this thread again.

Reply:
Update data explorer models in [public storage](https://storage.openvinotoolkit.org/), please check https://github.com/openvinotoolkit/datumaro/pull/813.